tests/search/search_tests.py

```diff
@@ -169,6 +176,52 @@ def test_search_terms(self):
         request.user = self.tester
         response = search_terms(request)
         result = {}
+        try:
+            result = json.loads(response.content)
+        except json.decoder.JSONDecodeError:
+            print("Failed to parse search result")
```

Comment:
What failure condition is being tested here? Maybe remove the except and let it fail so we know it's flaky? EDIT: oh, it's blending it with surrounding tests, I leave it to you then 👍